lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -92,6 +92,10 @@ def execute(command, options = {})
       def shellescape(string)
         Shellwords.escape(string)
       end
+
+      def qpid_tools_installed?
```

Comment:
No specific reason. 
I have given `QpidQueues` as a name of feature that's why I thought this might be needed for other features specific to qpid.
As per your above comment, I will change feature name as `qpid` and will move this method to feature itself.